src/assets/svg/icons/actions/Confirm.tsx

```diff
@@ -1,9 +1,15 @@
 import * as React from 'react';
 import Svg, {SvgProps, Path} from 'react-native-svg';
 
-function SvgConfirm(props: SvgProps) {
+function SvgConfirm(props: SvgProps, fill?: string) {
```

Comment:
SVG-er bør ikke endres direkte, vi bruker et script til å gjøre dette. Vi legger ikoner under `<root>/assets/svgs`, og så kjører vi `yarn generate-svgs` for å generere alle React-componentene for disse. Gjerne ta kontakt om uklart :D

Reply:
Hvis du må endre fill så skal det være mulig å gjøre dette fra utsiden av denne componenten